bugshaker/src/main/java/com/github/stkent/bugshaker/MainActivity.java

```diff
@@ -144,10 +175,25 @@ public void paintClicked(View view) {
 		}
 	}
 
-	private Uri getImageUri(Context inContext, Bitmap inImage) {
+	private void createTextEdit() {
+
+		View.OnTouchListener onTouchListener = new View.OnTouchListener() {
+			@Override
+			public boolean onTouch(View view, MotionEvent motionEvent) {
+				float x = motionEvent.getX();
+				float y = motionEvent.getY();
+				final EditText editText;
+				editText = new EditText(getApplicationContext());
```

Comment:
I think this is a leftover mistake since createTextEdit is never called, so we just deleted it